src/main/java/com/codecool/barbershop/barbershop/dashboard/DashboardService.java

```diff
@@ -29,7 +29,7 @@ public DashboardData getDataForDashBoard() {
         Date lastDayOfTheMonth = java.sql.Date.valueOf(LocalDate.now().with(TemporalAdjusters.lastDayOfMonth()));
 
         data.setReportDate(java.sql.Date.valueOf(LocalDate.now()));
-        data.setTotalClients(clientService.getTotalClients());
+        data.setNewClients(clientService.countNewClientsDateBetween(firstDayOfTheMonth,lastDayOfTheMonth));
         data.setTotalConfirmedBookings(bookingService.countBookingsByBookingDateBetweenAndBookingStatus(firstDayOfTheMonth, lastDayOfTheMonth, BookingStatus.CONFIRM));
         data.setTotalUpcomingBookings(bookingService.countBookingsByBookingDateBetweenAndBookingStatus(firstDayOfTheMonth, lastDayOfTheMonth, BookingStatus.UPCOMING));
         data.setTotalCanceledBookings(bookingService.countBookingsByBookingDateBetweenAndBookingStatus(firstDayOfTheMonth, lastDayOfTheMonth, BookingStatus.CANCEL));
```

Comment:
Since there might be changes in the statuses as time goes by (new ones added, old ones renamed), I'd rather aggregate those calls into an iteration over the statuses.  In general **for each** status execute appropriate method to get total counts of bookings.